cfme/control/explorer/policy_profiles.py

```diff
@@ -0,0 +1,182 @@
+        else:
+            return None
+
+    def update(self, updates):
+        """Update this Policy Profile in UI.
+
+        Args:
+            updates: Provided by update() context manager.
+            cancel: Whether to cancel the update (default False).
+        """
+        view = navigate_to(self, "Edit")
+        changed = view.fill(updates)
+        if changed:
+            view.save_button.click()
+        else:
+            view.cancel_button.click()
+        for attr, value in updates.items():
+            setattr(self, attr, value)
+        view = self.create_view(PolicyProfileDetailsView)
+        assert view.is_displayed
+        view.flash.assert_no_error()
+        if changed:
+            view.flash.assert_message(
+                'Policy Profile "{}" was saved'.format(
+                    updates.get("description", self.description)))
+        else:
+            view.flash.assert_message(
+                'Edit of Policy Profile "{}" was cancelled by the user'.format(self.description))
+
+    def delete(self, cancel=False):
+        """Delete this Policy Profile in UI.
+
+        Args:
+            cancel: Whether to cancel the deletion (default False).
+        """
+        view = navigate_to(self, "Details")
+        view.configuration.item_select("Remove this Policy Profile", handle_alert=not cancel)
+        if cancel:
+            assert view.is_displayed
+            view.flash.assert_no_error()
+        else:
+            view = self.create_view(PolicyProfilesAllView)
+            assert view.is_displayed
+            view.flash.assert_no_error()
+            view.flash.assert_message(
+                'Policy Profile "{}": Delete successful'.format(self.description))
+
+
+@navigator.register(PolicyProfile, "Add")
+class PolicyProfileNew(CFMENavigateStep):
+    VIEW = NewPolicyProfileView
+    prerequisite = NavigateToAttribute("appliance.server", "ControlExplorer")
+
+    def step(self):
+        self.view.policy_profiles.tree.click_path("All Policy Profiles")
+        # if self.view.policy_profiles.is_opened:
+        self.view.configuration.item_select("Add a New Policy Profile")
+
+
+@navigator.register(PolicyProfile, "Edit")
+class PolicyProfileEdit(CFMENavigateStep):
+    VIEW = EditPolicyProfileView
+    prerequisite = NavigateToAttribute("appliance.server", "ControlExplorer")
+
+    def step(self):
+        if self.view.policy_profiles.is_opened:
+            self.view.policy_profiles.tree.click_path("All Policy Profiles", self.obj.description)
+        self.view.configuration.item_select("Edit this Policy Profile")
+
+
+@navigator.register(PolicyProfile, "Details")
+class PolicyProfileDetails(CFMENavigateStep):
+    VIEW = PolicyProfileDetailsView
+    prerequisite = NavigateToAttribute("appliance.server", "ControlExplorer")
+
+    def step(self):
+        if self.view.policy_profiles.is_opened:
+            self.view.policy_profiles.tree.click_path("All Policy Profiles", self.obj.description)
```

Comment:
I told about it before. There is a bug or some incorrect behavior of widgetastic. Policy profile accordion is already opened when you navigate to it. Without this condition accordion will be closed.